components/dashboard/src/workspaces/WorkspaceStatusIndicator.tsx

```diff
@@ -4,31 +4,28 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { WorkspaceInstance, WorkspaceInstanceConditions, WorkspaceInstancePhase } from "@gitpod/gitpod-protocol";
+import { WorkspaceInstanceConditions } from "@gitpod/gitpod-protocol";
 import Tooltip from "../components/Tooltip";
+import { WorkspacePhase_Phase, WorkspaceStatus } from "@gitpod/public-api/lib/gitpod/v1/workspace_pb";
 
-export function WorkspaceStatusIndicator({ instance }: { instance?: WorkspaceInstance }) {
-    const state: WorkspaceInstancePhase = instance?.status?.phase || "stopped";
-    const conditions = instance?.status?.conditions;
+export function WorkspaceStatusIndicator({ status }: { status?: WorkspaceStatus }) {
+    const state: WorkspacePhase_Phase = status?.phase?.name || WorkspacePhase_Phase.STOPPED;
+    const conditions = status?.conditions;
     let stateClassName = "rounded-full w-3 h-3 text-sm align-middle";
     switch (state) {
-        case "running": {
+        case WorkspacePhase_Phase.RUNNING: {
             stateClassName += " bg-green-500";
             break;
         }
-        case "stopped": {
+        case WorkspacePhase_Phase.STOPPED: {
             if (conditions?.failed) {
                 stateClassName += " bg-red-400";
             } else {
                 stateClassName += " bg-gray-400";
             }
             break;
         }
-        case "interrupted": {
-            stateClassName += " bg-red-400";
-            break;
-        }
-        case "unknown": {
```

Comment:
this one is UNSPECIFIED?

Reply:
`const state: WorkspacePhase_Phase = status?.phase?.name || WorkspacePhase_Phase.STOPPED;`

state will never be `UNSPECIFIED`, so add this case ts server will show us an error